### DISCUSSION TITLE
Pro GitHub

### DISCUSSION THREAD

**Discussion by awalfalahi** [community/community#62519] [GitHub Education] [answered]
### Select Topic Area

General

### Body

I am student and i want to get GitHub pro

**Comment by awalfalahi** [comment]
I am student and i want to get GitHub pro

**Comment by awalfalahi** [comment]
I am student and  want to get GitHub pro

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @awalfalahi, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your ask entails. I've gone ahead and moved it for you. Good luck!

**Comment by 6ixfalls** [comment]
Sign in at https://education.github.com/pack/join, and verify as a student to gain access to GitHub's Student Developer Pack.

**Comment by Awrsha** [comment, marked as answer]
Here are the steps to get GitHub Pro as a student:

Make sure you meet the eligibility requirements:
You must be currently enrolled as a full time student at an accredited high school, college or university.

You will need to provide proof of enrollment like a student ID, registration documents, or tuition invoice.

Apply for the GitHub Student Developer Pack:
Go to github.com/education and click "Get started for free."

Sign in with your GitHub account or create a new one.

Follow the steps to submit your proof of enrollment.

Wait for your application to be approved. This may take 1-2 business days.

Redeem your GitHub Student Developer Pack benefits:
You'll receive an email with coupon codes for your benefits.

One of the benefits is a coupon for 1 year of GitHub Pro for free.

Redeem the GitHub Pro coupon on your GitHub billing page.

Activate and use GitHub Pro:
After redeeming the coupon, it may take 1-2 business days for GitHub Pro to activate on your account.

You'll then have access to all GitHub Pro features for 1 full year.

At the end of the year, you'll either need to upgrade to a paid GitHub Pro plan or your account will revert to the free GitHub Free plan.

So in summary, the key steps are to apply for the Student Developer Pack, redeem your benefits which includes 1 year of free GitHub Pro, and then activate and enjoy the GitHub Pro features for that 1 year.

**Comment by xlightningxstar** [comment]
Response #1
